oojs.js

```diff
@@ -10,3 +10,27 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+class Playlist {
+  constructor (title, songs) {
+    this.title = title;
+    this.songs = ["Workout", "Fun", "Zumba"];
+    this.favorites = 0;
+  }
+  addSong(songs) {
+    this.songs.push(songs);
+    console.log(this.songs);
+  }
```

Comment:
This works if you are trying to pass another array of songs into `Playlist`'s `songs` property. But the question is asking for a method that adds just one song to the `songs` property.